glsl_builders.py

```diff
@@ -127,10 +139,7 @@ def build_rd_header(filename):
     if len(header_data.compute_lines):
 
         fd.write("\t\tstatic const char _compute_code[] = {\n")
-        for x in header_data.compute_lines:
-            for c in x:
-                fd.write(str(ord(c)) + ",")
-            fd.write(str(ord("\n")) + ",")
+        fd.write(generate_inline(header_data.compute_lines))
```

Comment:
The rest won't make a big impact on speed, but I think it might make sense to go all the way have keep a single write for the whole `build_rd_header`.

We can likely make use of multiline strings and f-Strings (now that we require Python 3.6+ in `master`) to try to keep things somewhat readable while doing so.

This could be a follow-up PR though as it's more code quality than optimization.